FE/src/components/index.ts

```diff
@@ -16,6 +16,7 @@ import QuestionAnswerCard from './QuestionAnswerCard/QuestionAnswerCard';
 import QuestionAnswerFormCard from './QuestionAnswerFormCard/QuestionAnswerFormCard';
 import Scroller from './Scroller/Scroller';
 import Loading from './Loading/Loading';
+import { ProfileInfo } from './ProfileInfo/ProfileInfo';
```

Comment:
import 방식의 통일성을 지켜주세요
```suggestion
import ProfileInfo from './ProfileInfo/ProfileInfo';
```

Reply:
![image](https://github.com/boostcampwm2023/web04-ALGOCEAN/assets/87417773/3b50b069-7596-4ab6-81d8-595965535283)

좋은 의견입니다. 
하지만 반반이라 통일성을 지키려면 한쪽으로의 전체적인 수정이 필요할 것 같네요.